SimG4CMS/Forward/src/CastorSD.cc

```diff
@@ -582,15 +599,15 @@ uint32_t CastorSD::rotateUnitID(uint32_t unitID, G4Track* track, const CastorSho
   
   // Get 'track' phi:
   double   trackPhi = track->GetPosition().phi(); 
-  if(trackPhi<0) trackPhi += 2*M_PI ;
+  if(trackPhi<0) trackPhi += twopi;
```

Comment:
where is `twopi` defined?

Reply:
It is CLHEP constant, which is used already in the same class, no sense of using from two different sources.